spec/controllers/service_providers_controller_spec.rb

```diff
@@ -16,6 +16,39 @@
         content_type: 'image/svg+xml',
       }
     end
+    # let(:sp_with_whitespaces) {create(:service_provider, :with_whitespaces, user: user)}
```

Comment:
**nitpick (style):** Can we remove commented code?

Reply:
This feels passive aggressive coming from you. 😄